lib/Screens/home_screen.dart

```diff
@@ -635,7 +648,7 @@ class MainDrawer extends StatelessWidget {
                 backgroundColor: Colors.white,
               ),
               title: Text(
-                "Ananth Sai",
+                "ANANTH SAI",
```

Comment:
@EAsmithaShrree I think you forgot to remove hardcoded the name here, can u make it show the name from the FirebaseUser Instance.